test/com/puppetlabs/puppetdb/test/scf/storage.clj

```diff
@@ -1094,11 +1094,18 @@
     (is (= (map :name (query-to-vec "SELECT name FROM certnames ORDER BY name ASC"))
            ["node1" "node3"]))))
 
+(defn update-event-timestamps
+  "Changes each timestamp in the `report`'s resource-events to `new-timestamp`"
+  [report new-timestamp]
+  (update-in report [:resource-events]
+             (fn [events]
```

Comment:
this is just a straight backport of what we have in master, so I think any style concerns should be addressed there to ease the rollups.